lua/neotest/consumers/output.lua

```diff
@@ -171,7 +172,16 @@ function neotest.output.open(opts)
   end
   async.run(function()
     local tree, adapter_id
-    if not opts.position_id then
+    if opts.last_run then
+      local position_id, last_args = require("neotest").run.get_last_run()
```

Comment:
when doing a `require('neotest').run` on top of the file of a `neotest.run` here, I will get an error run being `nil`